PolishWithBlondes/src/components/navbar/navbar.scss

```diff
@@ -0,0 +1,42 @@
+.navbar {
+  background-color: var(--primary-background);
+  color: var(--primary-font-color);
+  font-family: var(--font-family);
+  font-size: 14px;
+  padding: 40px;
+  display: flex;
+  justify-content: space-between;
+  max-width: 1060px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.navbar ul {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.navbar li {
+  align-self: center;
+  margin: 0px 5px;
+}
+
+.navbar a {
+  color: var(--primary-font-color);
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.logo {
+  padding: 0;
```

Comment:
is it needed? Padding should be 0 thanks to reset.scss if not. set it to 0 for everything there

Reply:
Yes, that's true. Resolved